giving-example/src/main/resources/templates/result.html

```diff
@@ -0,0 +1,34 @@
+<html lang="en"
+      xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layout}">
+<head>
+  <title th:text="${type}"></title>
```

Comment:
We should rename this to `result`, but I see we use it already in the other samples. Maybe we can create a TechDebt backlog, I find it confusing.

Reply:
What should we rename to result, the parameter "type"?